I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/intro_perso.rpy", line 5, in script
    j "Oh! Une nouvelle à l'école."
  File "game/intro_perso.rpy", line 5, in script
    j "Oh! Une nouvelle à l'école."
AttributeError: 'NoneType' object has no attribute 'get_window_flags'

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "/home/Perrine/POP-ular-Attraction/renpy-8.3.4-sdk/renpy/bootstrap.py", line 359, in bootstrap
    renpy.main.main()
  File "/home/Perrine/POP-ular-Attraction/renpy-8.3.4-sdk/renpy/main.py", line 652, in main
    run(restart)
  File "/home/Perrine/POP-ular-Attraction/renpy-8.3.4-sdk/renpy/main.py", line 148, in run
    renpy.execution.run_context(True)
  File "/home/Perrine/POP-ular-Attraction/renpy-8.3.4-sdk/renpy/execution.py", line 958, in run_context
    context.run()
  File "game/intro_perso.rpy", line 5, in script
    j "Oh! Une nouvelle à l'école."
  File "lib/python3.9/future/utils/__init__.py", line 444, in raise_
  File "game/intro_perso.rpy", line 5, in script
    j "Oh! Une nouvelle à l'école."
  File "/home/Perrine/POP-ular-Attraction/renpy-8.3.4-sdk/renpy/ast.py", line 2579, in execute
    Say.execute(self)
  File "/home/Perrine/POP-ular-Attraction/renpy-8.3.4-sdk/renpy/ast.py", line 621, in execute
    renpy.exports.say(who, what, *args, **kwargs)
  File "/home/Perrine/POP-ular-Attraction/renpy-8.3.4-sdk/renpy/exports/sayexports.py", line 132, in say
    who(what, *args, **kwargs)
  File "/home/Perrine/POP-ular-Attraction/renpy-8.3.4-sdk/renpy/character.py", line 1455, in __call__
    self.do_display(who, what, cb_args=self.cb_args, dtt=dtt, **display_args)
  File "/home/Perrine/POP-ular-Attraction/renpy-8.3.4-sdk/renpy/character.py", line 1106, in do_display
    display_say(who,
  File "/home/Perrine/POP-ular-Attraction/renpy-8.3.4-sdk/renpy/character.py", line 803, in display_say
    rv = renpy.ui.interact(mouse='say', type=type, roll_forward=roll_forward)
  File "/home/Perrine/POP-ular-Attraction/renpy-8.3.4-sdk/renpy/ui.py", line 301, in interact
    rv = renpy.game.interface.interact(roll_forward=roll_forward, **kwargs)
  File "/home/Perrine/POP-ular-Attraction/renpy-8.3.4-sdk/renpy/display/core.py", line 2215, in interact
    repeat, rv = self.interact_core(preloads=preloads, trans_pause=trans_pause, pause=pause, pause_start=pause_start, pause_modal=pause_modal, **kwargs) # type: ignore
  File "/home/Perrine/POP-ular-Attraction/renpy-8.3.4-sdk/renpy/display/core.py", line 2854, in interact_core
    if renpy.display.draw.update(force=self.display_reset):
  File "gl2draw.pyx", line 659, in renpy.gl2.gl2draw.GL2Draw.update
AttributeError: 'NoneType' object has no attribute 'get_window_flags'

Linux-6.12.10-200.fc41.x86_64-x86_64-with-glibc2.40 x86_64
Ren'Py 8.3.4.24120703
Pop_ular Attraction 1.0
Sun Jan 26 16:38:16 2025
